features/old/stats/provider_side.feature

```diff
@@ -39,63 +40,3 @@ Feature: Provider stats
     And I follow "Analytics"
     And I follow "Top Applications"
     Then I should see "Top Applications" in a header
-
-
-
-  @wip
```

Comment:
IMO, as a separate issue to this one, we should remove all the @wip cukes and skipped tests because they have been like that for ages and they only occupy space while we debug... 😄

Reply:
yep, we should remove them all.

Reply:
[THREESCALE-5471](https://issues.redhat.com/browse/THREESCALE-5471) 😎